fizzbuzz.js

```diff
@@ -10,12 +10,31 @@ function arrayEquals(arr1, arr2) {
 /**
  * PART 0
  *
- * Write a function that takes two numbers as
- * arguments and computes the sum of those two numbers.
+  Write a function that takes two numbers as* arguments and computes the sum of those two numbers.
  */
 
+function denominators (num) {
+	var denoms = [];
+
+for (var x = 1; x <= num; x++){
+	if (num % x === 0 ) {
+		return denoms
+}
+
+
+if (num==1){
+	return [1];
+}	return[ 1,num]; // body...
+ return denoms
+}
+console.assert(arrayEquals(denominators(1), [1]));
+console.assert(arrayEquals(denominators(2), [1, 2]));
+console.assert(arrayEquals(denominators(3), [1, 3]));
+console.assert(arrayEquals(denominators(4), {1, 2, 4}
+console.assert(arrayEquals(denominators(24), {1, 2, 3, 4,6, 24]));
```

Comment:
This is throwing an error and I cannot test the rest of the file. Check your syntax. You have a `{` where you should have `[` for an array.